cmd/kubelet/app/server.go

```diff
@@ -338,7 +343,7 @@ type KubeletConfig struct {
 	EtcdClient                     tools.EtcdClient
 	KubeClient                     *client.Client
 	DockerClient                   dockertools.DockerInterface
-	CAdvisorPort                   uint
+	CAdvisorInterface              cadvisor.Interface
```

Comment:
nit: `CAdvisor` is a slightly confusing naming pattern. Can we change it to `Cadvisor'?